### PR TITLE
benetech/ServiceNet#1141: Fix record cards placement when filtering

### PR DESCRIPTION
**Pull request by pmuchowski**

### CODE REVIEW

File: src/main/webapp/app/modules/provider/all-records.tsx [hunk 1]
```diff
@@ -296,16 +300,27 @@ export class AllRecords extends React.Component<IAllRecordsProps, IAllRecordsSta
     const { filterOpened, isMapView } = this.state;
     const { siloName } = this.props;
     const elementsBesideFilter = _.slice(allRecords, 0, 4);
+    const elementsBesideFilterFirstPart = _.slice(allRecords, 0, 2);
```
Comment (jkondrat) [inline src/main/webapp/app/modules/provider/all-records.tsx]: what does the first and second part stand for?

Comment (pmuchowski) [reply src/main/webapp/app/modules/provider/all-records.tsx]: Depending on the screen size you will display 2 or 4 record cards beside the filter, so if only 2 are displayed beside the filter the elementsBesideFilterFirstPart will be used and elementsBesideFilterSecondPart will be rendered under the filter and on large screens all 4 will be displayed beside the filter and elementsBesideFilter will be used. The elementsBesideFilter is split into 2 separate arrays and I just couldn't came up with better names for those arrays

Comment (jkondrat) [reply src/main/webapp/app/modules/provider/all-records.tsx]: how about:
fourCardsBesideFilter
upperTwoCardsBesideFilter
lowerTwoCardsBesideFilter

Comment (pmuchowski) [reply src/main/webapp/app/modules/provider/all-records.tsx]: sure, I will rename it and push the changes

**Pull request merged**